clients/src/main/java/org/apache/kafka/clients/producer/internals/Sender.java

```diff
@@ -216,7 +216,6 @@ void run(long now) {
         }
 
         long pollTimeout = sendProducerData(now);
-        log.trace("waiting {}ms in poll", pollTimeout);
```

Comment:
Yes, I found them a bit useless and distracting. I think @apurvam added a couple of these, so we can see if he had a compelling need.